articles/aks/deploy-spinkube.md

```diff
@@ -0,0 +1,253 @@
+
+```bash
+kubectl apply -f https://github.com/spinkube/spin-operator/releases/download/v0.2.0/spin-operator.shim-executor.yaml
+```
+
+## Run a Spin App on AKS
+
+In this section you will verify the SpinKube installation, by creating a simple Spin App using the `spin` CLI and JavaScript. Start by creating a new Spin App using the `http-js` template:
+
+```bash
+# Create a new Spin App
+spin new -t http-js --accept-defaults hello-spinkube
+
+# Move into the application folder
+cd hello-spinkube
+
+# Install Wasm Tooling
+npm install
+```
+
+The `spin` CLI create a basic *Hello, World* application, package and push the Spin App without further modifications to a OCI compliant container registry:
+
+```bash
+# Compile the app to Wasm
+spin build
+
+# Package and Distribute the Spin App
+spin registry push ttl.sh/hello-spinkube:0.0.1
+```
+
+> Both `spin` CLI and SpinKube integrate seamlessly with private container registries such as Azure Container Registry (ACR). To authenticate the `spin` CLI use `spin registry login` and supply corresponding credentials. SpinKube can pull OCI artifacts from ACR using underlying Azure identities with `AcrPull` permissions for the desired ACR instance(s).
+
+By using the `spin kube scaffold` command, you can create necessary Kubernetes deployment manifests. You can deploy those to Kubernetes using your preferred tooling:
+
+```bash
+# Create Kubernetes Deployment Manifests
+spin kube scaffold --from ttl.sh/hello-spinkube:0.0.1 > spinapp.yaml
+
+# Deploy the Spin App to AKS
+kubectl apply -f spinapp.yaml
+```
+
+### Explore the Spin App in AKS
+
+Having an Spin App deployed to the AKS cluster, you can explore different objects being created. 
+
+You can retrieve the list of Spin Apps using `kubectl get spinapps`
+
+```bash
+# Get all Spin Apps in the default namespace
+kubectl get spinapps
+NAME             READY   DESIRED   EXECUTOR
+hello-spinkube   2       2         containerd-shim-spin
+```
+
+Upon deployment, the `spin-operator` creates underlying Kubernetes primitives such as a *Service*, a *Deployment* and corresponding *Pods*:
+
+```bash
+# Retrieve Kubernetes primitives created by the spin-operator
+kubectl get service
+NAME             TYPE        CLUSTER-IP    EXTERNAL-IP   PORT(S)   AGE
+hello-spinkube   ClusterIP   10.43.35.78   <none>        80/TCP    24s
+
+
+kubectl get deployment
+NAME             READY   UP-TO-DATE   AVAILABLE   AGE
+hello-spinkube   2/2     2            2           38s
+
+
+kubectl get pod
+NAME                              READY   STATUS    RESTARTS   AGE
+hello-spinkube-5b8579448d-zmc6x   1/1     Running   0          51s
+hello-spinkube-5b8579448d-bhkp9   1/1     Running   0          51s
+```
+
+To invoke the Spin App, you configure port-forwarding to the service provisioned by the `spin-operator` and use `curl` for sending HTTP requests:
+
+```bash
+# Establish port forwarding
+kubectl port-forward svc/hello-spinkube 8080:80
+Forwarding from 127.0.0.1:8080 -> 80
```

Comment:
Please consider splitting output from executable command.

Reply:
Addressed as part of 4d6a07f